docs/installing/preparing_edge_cluster.md

```diff
@@ -281,11 +282,11 @@ This content provides a summary of how to install microk8s, a lightweight and sm
       ```
       {: codeblock}
 
-   3. Define this registry as insecure to docker. Create or add to **/etc/docker/daemon.json** (replacing `<registry-endpoint>` with the value of the `$REGISTRY_ENDPOINT` environment variable you obtained in a previous step).
+   3. Define this registry as insecure to docker. Create or add to **/etc/docker/daemon.json** (replacing `<registry-endpoint>` with the value of the `$REGISTRY_ENDPOINT` environment variable you obtained in a previous step. Replacing `<registry-ip-endpoint>` with the value of the `$REGISTRY_IP_ENDPOINT` environment variable you obtained in a previous step).
```

Comment:
It looks like this sentence or phrase is repeated (the one starting with the work replacing). Did you mean to replace the original one w/the update? I can not sure, so can you check? TY!!

(replacing `<registry-endpoint>` with the value of the `$REGISTRY_ENDPOINT` environment variable you obtained in a previous step. Replacing `<registry-ip-endpoint>` with the value of the `$REGISTRY_IP_ENDPOINT` environment variable you obtained in a previous step).